ui/components/ui/tabs/tabs.tsx

```diff
@@ -86,32 +84,19 @@ export const Tabs = <TKey extends string = string>({
   };
 
   return (
-    <Box
-      className={twMerge('tabs', 'transform-gpu', props.className)}
-      {...props}
-    >
+    <Box className={twMerge('tabs', 'transform-gpu', className)} {...props}>
```

Comment:
Keeping the `tabs` className even though no styles attached for an easy identifier when debugging and searching via dev tools